### ISSUE TITLE
FTBFS on Debian testing/sid

### ISSUE THREAD

**Issue by sunweaver** [ArcticaProject/nx-libs#1021]
```
Building Release 3.5.99.26.

Mon Aug 30 08:10:56 UTC 2021

make -f xmakefile -w -j1  World
make[5]: Entering directory '/<<PKGBUILDDIR>>/nx-X11'
make -w -j1 -f xmakefile  all
make[6]: Entering directory '/<<PKGBUILDDIR>>/nx-X11'
making all in ./include...
make[7]: Entering directory '/<<PKGBUILDDIR>>/nx-X11/include'
making all in include/extensions...
make[8]: Entering directory '/<<PKGBUILDDIR>>/nx-X11/include/extensions'
make[8]: Nothing to be done for 'all'.
make[8]: Leaving directory '/<<PKGBUILDDIR>>/nx-X11/include/extensions'
making all in include/GL...
make[8]: Entering directory '/<<PKGBUILDDIR>>/nx-X11/include/GL'
make[8]: Nothing to be done for 'all'.
make[8]: Leaving directory '/<<PKGBUILDDIR>>/nx-X11/include/GL'
make[7]: Leaving directory '/<<PKGBUILDDIR>>/nx-X11/include'
making all in ./config...
make[7]: Entering directory '/<<PKGBUILDDIR>>/nx-X11/config'
making all in config/cf...
make[8]: Entering directory '/<<PKGBUILDDIR>>/nx-X11/config/cf'
make[8]: Nothing to be done for 'all'.
make[8]: Leaving directory '/<<PKGBUILDDIR>>/nx-X11/config/cf'
make[7]: Leaving directory '/<<PKGBUILDDIR>>/nx-X11/config'
making all in ./programs...
make[7]: Entering directory '/<<PKGBUILDDIR>>/nx-X11/programs'
making all in programs/Xserver...
make[8]: Entering directory '/<<PKGBUILDDIR>>/nx-X11/programs/Xserver'
making all in programs/Xserver/include...
make[9]: Entering directory '/<<PKGBUILDDIR>>/nx-X11/programs/Xserver/include'
make[9]: Nothing to be done for 'all'.
make[9]: Leaving directory '/<<PKGBUILDDIR>>/nx-X11/programs/Xserver/include'
making all in programs/Xserver/dix...
make[9]: Entering directory '/<<PKGBUILDDIR>>/nx-X11/programs/Xserver/dix'
make[9]: Nothing to be done for 'all'.
make[9]: Leaving directory '/<<PKGBUILDDIR>>/nx-X11/programs/Xserver/dix'
making all in programs/Xserver/os...
make[9]: Entering directory '/<<PKGBUILDDIR>>/nx-X11/programs/Xserver/os'
  CC rpcauth.c
rpcauth.c:47:10: fatal error: rpc/rpc.h: No such file or directory
   47 | #include <rpc/rpc.h>
      |          ^~~~~~~~~~~
compilation terminated.
failed command: gcc -c -Wdate-time -D_FORTIFY_SOURCE=2 -g -O2 -ffile-prefix-map=/<<PKGBUILDDIR>>=. -fstack-protector-strong -Wformat -Werror=format-security -std=c99 -Wall -Wpedantic -Wpointer-arith -Wshadow -Wundef -I. -I../include -I../../../exports/include/nx-X11 -I../../../include/extensions -I../../../programs/Xserver/Xext -I../../../programs/Xserver/render -I../../../lib/Xau -I../../../../nxcomp -I/usr/include/pixman-1 -I../../.. -I../../../exports/include -Dlinux -D__amd64__ -D_POSIX_C_SOURCE=199309L -D_POSIX_SOURCE -D_XOPEN_SOURCE -D_DEFAULT_SOURCE -D_LARGEFILE_SOURCE -D_FILE_OFFSET_BITS=64 -D_GNU_SOURCE -DSHAPE -DXINPUT -DXKB -DXCSECURITY -DXF86BIGFONT -DDPMSExtension -DRENDER -DRANDR -DPANORAMIX -DXFIXES -DDAMAGE -DCOMPOSITE -DGCCUSESGAS -DAVOID_GLYPHBLT -DPIXPRIV -DSINGLEDEPTH -DXvExtension -DXFree86Server -DXvMCExtension -DBUILDDEBUG -DXResExtension -DX_BYTE_ORDER=X_LITTLE_ENDIAN -DXORG_VERSION_CURRENT=(((6) * 10000000) + ((9) * 100000) + ((0) * 1000) + 0) -D_XSERVER64 -DNDEBUG -DHAVE_SETITIMER=1 -DHAS_XFONT2 -DFUNCPROTO=15 -DNARROWPROTO -DUNIXCONN -DTCPCONN -DHAS_STICKY_DIR_BIT -DHAS_FCHOWN -DIPv6 -DHASXDMAUTH -DSECURE_RPC -DDDXOSINIT -DSERVER_LOCK -DDDXOSFATALERROR -DDDXOSVERRORF -DCLIENTIDS -DBUILDERADDR="xorg@lists.freedesktop.org" -DFAIL_HARD -DXTRANS_SEND_FDS=0 -DNX_TRANS_SOCKET -DNX_TRANS_AUTH -DNX_TRANS_FOPEN -DNX_TRANS_SLEEP -DNX_TRANS_EXIT -DNX_TRANS_WAKEUP -DNXAGENT_SERVER rpcauth.c
make[9]: *** [Makefile:531: rpcauth.o] Error 1
make[9]: Leaving directory '/<<PKGBUILDDIR>>/nx-X11/programs/Xserver/os'
make[8]: *** [Makefile:703: os] Error 2
make[8]: Leaving directory '/<<PKGBUILDDIR>>/nx-X11/programs/Xserver'
make[7]: *** [Makefile:463: all] Error 2
make[7]: Leaving directory '/<<PKGBUILDDIR>>/nx-X11/programs'
make[6]: *** [xmakefile:463: all] Error 2
make[6]: Leaving directory '/<<PKGBUILDDIR>>/nx-X11'
make[5]: *** [xmakefile:524: World] Error 2
make[5]: Leaving directory '/<<PKGBUILDDIR>>/nx-X11'
make[4]: *** [Makefile:42: World] Error 2
make[4]: Leaving directory '/<<PKGBUILDDIR>>/nx-X11'
make[3]: *** [Makefile:179: build-full] Error 2
make[3]: Leaving directory '/<<PKGBUILDDIR>>'
make[2]: *** [Makefile:191: build] Error 2
make[2]: Leaving directory '/<<PKGBUILDDIR>>'
dh_auto_build: error: make -j1 "CDEBUGFLAGS=-Wdate-time -D_FORTIFY_SOURCE=2 -g -O2 -ffile-prefix-map=/<<PKGBUILDDIR>>=. -fstack-protector-strong -Wformat -Werror=format-security" "LOCAL_LDFLAGS=-Wl,-z,relro -Wl,-z,now" "SHLIBGLOBALSFLAGS=-Wl,-z,relro -Wl,-z,now" returned exit code 2
make[1]: *** [debian/rules:62: override_dh_auto_build] Error 25
make[1]: Leaving directory '/<<PKGBUILDDIR>>'
make: *** [debian/rules:16: build] Error 2
```

**Comment by sunweaver** [comment]
See https://jenkins.arctica-project.org/job/nx-libs.deb+nightly/

**Comment by uli42** [comment]
a) you need tirpc
b) I had the same on a recent RHEL8 compilation and had to use
IMAKE_DEFINES="-DUseTIRPC=YES"

Uli

**Comment by uli42** [comment]
Your fix might work for a regular debian build but it does not work for the github action on SID. see .github/workflows/nx-libs.yml.

**Comment by sunweaver** [comment]
Resolved by #1026.
Resolved by b67b910a6adf3287fad10cc97cf9484b0057b0bd and 6be4f87c63fe8c816dac4dbd0ed53727ecad0f4d.
